### PR TITLE
Xamarin oidcclient

### PR DESCRIPTION
**Pull request by MarcosCobena**
Hello there,

I've updated the endpoints in the mobile scenario as well as fixed UWP login, which was broken due to a change in WebView's events.

Also tested the app in Android, iOS and UWP with login, basket fulfill and checkout, and every platform seems to work as expected.

—Marcos

### CODE REVIEW

File: src/Mobile/eShopOnContainers/eShopOnContainers.Core/GlobalSettings.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@ public class GlobalSetting
     {
         public const string AzureTag = "Azure";
         public const string MockTag = "Mock";
-        public const string DefaultEndpoint = "http://13.88.8.119";
+        public const string DefaultEndpoint = "http://52.170.199.82";
```
Comment (jsuarezruiz) [inline src/Mobile/eShopOnContainers/eShopOnContainers.Core/GlobalSettings.cs]: This is a temporal endpoint. Change this for something like "INSERT YOUR ENDPOINT".

Comment (MarcosCobena) [reply src/Mobile/eShopOnContainers/eShopOnContainers.Core/GlobalSettings.cs]: Done!

**Comment by jsuarezruiz** [comment]
Thanks for the change ;). Let me include @davidbritch to review this.

**Comment by CESARDELATORRE** [comment]
The only issue with "INSERT YOUR ENDPOINT" is that it is not very clear what kind. IP? DNS? Both work? Does it need to include the "http://" prefix?
I'd add a comment with two examples, like this comment:
// i.e.: "http://YOUR_IP"  or  "http://YOUR_DNS_NAME"
Which clearly tells you that you need to include the "http://" prefix.

**Comment by MarcosCobena** [comment]
Hey @CESARDELATORRE:

Following your comment I think this works even better in those cases where people clone the repo and hit Run directly, so won't notice the GlobalSettings.cs file in advance:

image

Thanks!

—Marcos

**Comment by CESARDELATORRE** [comment]
@MarcosCobena Cool! - Let's do both? 👍

File: src/Mobile/eShopOnContainers/eShopOnContainers.Core/Services/Catalog/CatalogService.cs [hunk 2]
```diff
@@ -22,8 +24,8 @@ public CatalogService(IRequestProvider requestProvider, IFixUriService fixUriSer
 
         public async Task<ObservableCollection<CatalogItem>> FilterAsync(int catalogBrandId, int catalogTypeId)
         {
-            UriBuilder builder = new UriBuilder(GlobalSetting.Instance.CatalogEndpoint);
-            builder.Path = string.Format("api/v1/catalog/items/type/{0}/brand/{1}", catalogTypeId, catalogBrandId);
+            UriBuilder builder = new UriBuilder(GlobalSetting.Instance.BaseEndpoint);
+            builder.Path = string.Format($"{ApiUrlBase}/items/type/{0}/brand/{1}", catalogTypeId, catalogBrandId);
```
[the remark this answers is not in the document]
Comment (MarcosCobena) [reply src/Mobile/eShopOnContainers/eShopOnContainers.Core/Services/Catalog/CatalogService.cs]: Good catch, agree too, done!

File: src/Mobile/eShopOnContainers/eShopOnContainers.Core/Services/Order/OrderService.cs [hunk 3]
```diff
@@ -40,9 +42,9 @@ public Task CreateOrderAsync(Models.Orders.Order newOrder, string token)
         {
             try
             {
-                UriBuilder builder = new UriBuilder(GlobalSetting.Instance.OrdersEndpoint);
+                UriBuilder builder = new UriBuilder(GlobalSetting.Instance.BaseEndpoint);
 
-                builder.Path = string.Format("api/v1/orders/{0}", orderId);
+                builder.Path = string.Format($"{ApiUrlBase}/{0}", orderId);
```
[the remark this answers is not in the document]
Comment (MarcosCobena) [reply src/Mobile/eShopOnContainers/eShopOnContainers.Core/Services/Order/OrderService.cs]: Done!

**Comment by davidbritch** [comment]
Thanks @MarcosCobena :)

**Comment by MarcosCobena** [comment]
Welcome! :-)

**Pull request merged**